app/vmui/packages/vmui/src/pages/ExploreAnomaly/ExploreAnomaly.tsx

```diff
@@ -87,7 +87,7 @@ const ExploreAnomaly: FC = () => {
         setHideError={setHideError}
         stats={queryStats}
         onRunQuery={handleRunQuery}
-        hideButtons={{ addQuery: true, prettify: true, autocomplete: true, traceQuery: true, anomalyConfig: true }}
+        hideButtons={{ addQuery: true, prettify: false, autocomplete: false, traceQuery: true, anomalyConfig: true }}
```

Comment:
That was probably hidden when anomaly selected between predefined query-keys, not needed anymore.
Should I delete the option to hide query/prettify/autocomplete? I don't think those options are used anywhere else.

(Actually not sure about whole hideButtons as well)

Reply:
Previously, the API did not support `prettify` and `autocomplete` in vmanomaly. 
The `addQuery` displays an option to add multiple expression input fields.